src/cli.rs

```diff
@@ -91,6 +91,9 @@ pub struct Cli {
     /// formatting.
     #[arg(long)]
     pub line_length: Option<usize>,
+    /// Max McCabe complexity allowed for a function.
+    #[arg(long)]
+    pub max_complexity: Option<isize>,
```

Comment:
Could we make this `usize` everywhere?

Reply:
Sure, my hesitance for that came from the default in `mccabe` being `-1`. If we default to something else (e.g. `10`) then `usize` is good.